### ISSUE TITLE
is v1.8.2 in the repository? 

### ISSUE THREAD

**Issue by jirikrepl** [TAPevents/tap-i18n#215]
On atmosphere version 1.8.2 is available

Is the tip of master branch actually version 1.8.2?

The last tag in the repository is 1.8.1

**Comment by theosp** [comment]
**TL;DR you can safely work on v1.8.1 .**

Needless to say it is a big fail it isn't here (and I have it no-where). Since so long time passed, I don't even have a clue what caused 24 hours after the release of v1.8.1 to release v1.8.2 . 

*But*

We can tell the what is the difference and resurrect v1.8.2 source code with a little effort:

v1.8.1 is on atmosphere. When meteor downloads packages from atmosphere it downloads them to your: ~/.meteor/packages/ .

Right now, I have the following on my ~/.meteor/packages/tap_i18n:

1.6.1 , 1.8.2

All that is needed in order to find the diff between v1.8.2 to v1.8.1 is to create an app that will request specifically v1.8.2 ...

$ meteor create x
$ cd x
$ meteor add tap:i18n@=1.8.1

And now I have it under: ~/.meteor/packages/tap_i18n/1.8.1 :)

$ cd ~
$ mkdir temp
$ cp -r ~/.meteor/packages/tap_i18n/1.8.1 temp
$ cd temp/1.8.1
$ git init # Out of laziness
$ git add .
$ git commit -m v1.8.1
$ cd ..
$ cp -r ~/.meteor/packages/tap_i18n/1.8.2 .
$ mv 1.8.1/.git 1.8.2

$ rm -rf *
$ cp -r ~/.meteor/packages/tap_i18n/1.8.2/* .

$ git status

```
On branch master
Changes not staged for commit:
  (use "git add/rm <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

	modified:   isopack.json
	modified:   os.json
	deleted:    os/home/theosp/tap-i18n/unittest/test-packages/blank-package-tap-i18n/package-tap.i18n
	deleted:    os/home/theosp/tap-i18n/unittest/test-packages/blank-yaml-package-tap-i18n/package-tap.i18n
	deleted:    os/home/theosp/tap-i18n/unittest/test-packages/custom-package-tap-i18n/package-tap.i18n
	deleted:    os/home/theosp/tap-i18n/unittest/test-packages/empty-object-package-tap-i18n/package-tap.i18n
	deleted:    os/home/theosp/tap-i18n/unittest/test-packages/pack-with-project-trans-tap-i18n/package-tap.i18n
	deleted:    os/home/theosp/tap-i18n/unittest/test-packages/project-preload-all-langs/project-tap.i18n
	deleted:    os/packages/tap_i18n/packages/tap_i18nunittest/test-lang-files/bb.i18n.json
	deleted:    os/packages/tap_i18n/packages/tap_i18nunittest/test-lang-files/cc-CC.i18n.json
	deleted:    os/packages/tap_i18n/packages/tap_i18nunittest/test-lang-files/cc.i18n.json
	deleted:    os/packages/tap_i18n/packages/tap_i18nunittest/test-lang-files/dd.i18n.json
	deleted:    os/packages/tap_i18n/packages/tap_i18nunittest/test-lang-files/file1.en.i18n.json
	deleted:    os/packages/tap_i18n/packages/tap_i18nunittest/test-lang-files/file2.en.i18n.json
	deleted:    os/packages/tap_i18n/packages/tap_i18nunittest/test-packages/blank-package-tap-i18n/i18n/bb.i18n.json
	deleted:    os/packages/tap_i18n/packages/tap_i18nunittest/test-packages/blank-package-tap-i18n/i18n/cc-CC.i18n.json
	deleted:    os/packages/tap_i18n/packages/tap_i18nunittest/test-packages/blank-package-tap-i18n/i18n/cc.i18n.json
	deleted:    os/packages/tap_i18n/packages/tap_i18nunittest/test-packages/blank-package-tap-i18n/i18n/file1.en.i18n.json
	deleted:    os/packages/tap_i18n/packages/tap_i18nunittest/test-packages/blank-package-tap-i18n/i18n/file2.en.i18n.json
	deleted:    os/packages/tap_i18n/packages/tap_i18nunittest/test-packages/blank-yaml-package-tap-i18n/i18n/bb.i18n.yml
	deleted:    os/packages/tap_i18n/packages/tap_i18nunittest/test-packages/blank-yaml-package-tap-i18n/i18n/cc-CC.i18n.yml
	deleted:    os/packages/tap_i18n/packages/tap_i18nunittest/test-packages/blank-yaml-package-tap-i18n/i18n/cc.i18n.yml
	deleted:    os/packages/tap_i18n/packages/tap_i18nunittest/test-packages/blank-yaml-package-tap-i18n/i18n/file1.en.i18n.yml
	deleted:    os/packages/tap_i18n/packages/tap_i18nunittest/test-packages/blank-yaml-package-tap-i18n/i18n/file2.en.i18n.yml
	deleted:    os/packages/tap_i18n/packages/tap_i18nunittest/test-packages/custom-package-tap-i18n/i18n/bb.i18n.json
	deleted:    os/packages/tap_i18n/packages/tap_i18nunittest/test-packages/custom-package-tap-i18n/i18n/cc-CC.i18n.json
	deleted:    os/packages/tap_i18n/packages/tap_i18nunittest/test-packages/custom-package-tap-i18n/i18n/cc.i18n.json
	deleted:    os/packages/tap_i18n/packages/tap_i18nunittest/test-packages/custom-package-tap-i18n/i18n/file1.en.i18n.json
	deleted:    os/packages/tap_i18n/packages/tap_i18nunittest/test-packages/custom-package-tap-i18n/i18n/file2.en.i18n.json
	deleted:    os/packages/tap_i18n/packages/tap_i18nunittest/test-packages/empty-object-package-tap-i18n/i18n/bb.i18n.json
	deleted:    os/packages/tap_i18n/packages/tap_i18nunittest/test-packages/empty-object-package-tap-i18n/i18n/cc-CC.i18n.json
	deleted:    os/packages/tap_i18n/packages/tap_i18nunittest/test-packages/empty-object-package-tap-i18n/i18n/cc.i18n.json
	deleted:    os/packages/tap_i18n/packages/tap_i18nunittest/test-packages/empty-object-package-tap-i18n/i18n/file1.en.i18n.json
	deleted:    os/packages/tap_i18n/packages/tap_i18nunittest/test-packages/empty-object-package-tap-i18n/i18n/file2.en.i18n.json
	deleted:    os/packages/tap_i18n/packages/tap_i18nunittest/test-packages/pack-with-project-trans-tap-i18n/i18n/bb.i18n.json
	deleted:    os/packages/tap_i18n/packages/tap_i18nunittest/test-packages/pack-with-project-trans-tap-i18n/i18n/cc-CC.i18n.json
	deleted:    os/packages/tap_i18n/packages/tap_i18nunittest/test-packages/pack-with-project-trans-tap-i18n/i18n/cc.i18n.json
	deleted:    os/packages/tap_i18n/packages/tap_i18nunittest/test-packages/pack-with-project-trans-tap-i18n/i18n/file1.en.i18n.json
	deleted:    os/packages/tap_i18n/packages/tap_i18nunittest/test-packages/pack-with-project-trans-tap-i18n/i18n/file2.en.i18n.json
	deleted:    os/packages/tap_i18n/packages/tap_i18nunittest/test-packages/pack-with-project-trans-tap-i18n/i18n/file3.en.i18n.json
	deleted:    os/packages/tap_i18n/packages/tap_i18nunittest/test-packages/project-all-custom-project-tap-i18n/i18n/bb.i18n.json
	deleted:    os/packages/tap_i18n/packages/tap_i18nunittest/test-packages/project-all-custom-project-tap-i18n/i18n/cc-CC.i18n.json
	deleted:    os/packages/tap_i18n/packages/tap_i18nunittest/test-packages/project-all-custom-project-tap-i18n/i18n/file1.en.i18n.json
	deleted:    os/packages/tap_i18n/packages/tap_i18nunittest/test-packages/project-all-custom-project-tap-i18n/i18n/file2.en.i18n.json
	deleted:    os/packages/tap_i18n/packages/tap_i18nunittest/test-packages/project-no-project-tap-i18n/i18n/bb.i18n.json
	deleted:    os/packages/tap_i18n/packages/tap_i18nunittest/test-packages/project-no-project-tap-i18n/i18n/cc-CC.i18n.json
	deleted:    os/packages/tap_i18n/packages/tap_i18nunittest/test-packages/project-no-project-tap-i18n/i18n/file1.en.i18n.json
	deleted:    os/packages/tap_i18n/packages/tap_i18nunittest/test-packages/project-no-project-tap-i18n/i18n/file2.en.i18n.json
	deleted:    os/packages/tap_i18n/packages/tap_i18nunittest/test-packages/project-preload-all-langs/i18n/bb.i18n.json
	deleted:    os/packages/tap_i18n/packages/tap_i18nunittest/test-packages/project-preload-all-langs/i18n/cc-CC.i18n.json
	deleted:    os/packages/tap_i18n/packages/tap_i18nunittest/test-packages/project-preload-all-langs/i18n/file1.en.i18n.json
	deleted:    os/packages/tap_i18n/packages/tap_i18nunittest/test-packages/project-preload-all-langs/i18n/file2.en.i18n.json
	deleted:    os/packages/tap_i18n/packages/tap_i18nunittest/test-packages/project-with-translation-files-under-server-dir-SHOULD-FAIL/server/bb.i18n.json
	deleted:    os/packages/tap_i18n/packages/tap_i18nunittest/test-packages/project-with-translation-files-under-server-dir-SHOULD-FAIL/server/cc-CC.i18n.json
	deleted:    os/packages/tap_i18n/packages/tap_i18nunittest/test-packages/project-with-translation-files-under-server-dir-SHOULD-FAIL/server/file2.en.i18n.json
	deleted:    os/packages/tap_i18n/project-i18n.js
	modified:   web.browser.json
	deleted:    web.browser/home/theosp/tap-i18n/unittest/test-packages/blank-package-tap-i18n/package-tap.i18n
	deleted:    web.browser/home/theosp/tap-i18n/unittest/test-packages/blank-yaml-package-tap-i18n/package-tap.i18n
	deleted:    web.browser/home/theosp/tap-i18n/unittest/test-packages/custom-package-tap-i18n/package-tap.i18n
	deleted:    web.browser/home/theosp/tap-i18n/unittest/test-packages/empty-object-package-tap-i18n/package-tap.i18n
	deleted:    web.browser/home/theosp/tap-i18n/unittest/test-packages/pack-with-project-trans-tap-i18n/package-tap.i18n
	deleted:    web.browser/home/theosp/tap-i18n/unittest/test-packages/project-preload-all-langs/project-tap.i18n
	deleted:    web.browser/packages/tap_i18n/packages/tap_i18nunittest/test-lang-files/bb.i18n.json
	deleted:    web.browser/packages/tap_i18n/packages/tap_i18nunittest/test-lang-files/cc-CC.i18n.json
	deleted:    web.browser/packages/tap_i18n/packages/tap_i18nunittest/test-lang-files/cc.i18n.json
	deleted:    web.browser/packages/tap_i18n/packages/tap_i18nunittest/test-lang-files/dd.i18n.json
	deleted:    web.browser/packages/tap_i18n/packages/tap_i18nunittest/test-lang-files/file1.en.i18n.json
	deleted:    web.browser/packages/tap_i18n/packages/tap_i18nunittest/test-lang-files/file2.en.i18n.json
	deleted:    web.browser/packages/tap_i18n/packages/tap_i18nunittest/test-packages/blank-package-tap-i18n/i18n/bb.i18n.json
	deleted:    web.browser/packages/tap_i18n/packages/tap_i18nunittest/test-packages/blank-package-tap-i18n/i18n/cc-CC.i18n.json
	deleted:    web.browser/packages/tap_i18n/packages/tap_i18nunittest/test-packages/blank-package-tap-i18n/i18n/cc.i18n.json
	deleted:    web.browser/packages/tap_i18n/packages/tap_i18nunittest/test-packages/blank-package-tap-i18n/i18n/file1.en.i18n.json
	deleted:    web.browser/packages/tap_i18n/packages/tap_i18nunittest/test-packages/blank-package-tap-i18n/i18n/file2.en.i18n.json
	deleted:    web.browser/packages/tap_i18n/packages/tap_i18nunittest/test-packages/blank-yaml-package-tap-i18n/i18n/bb.i18n.yml
	deleted:    web.browser/packages/tap_i18n/packages/tap_i18nunittest/test-packages/blank-yaml-package-tap-i18n/i18n/cc-CC.i18n.yml
	deleted:    web.browser/packages/tap_i18n/packages/tap_i18nunittest/test-packages/blank-yaml-package-tap-i18n/i18n/cc.i18n.yml
	deleted:    web.browser/packages/tap_i18n/packages/tap_i18nunittest/test-packages/blank-yaml-package-tap-i18n/i18n/file1.en.i18n.yml
	deleted:    web.browser/packages/tap_i18n/packages/tap_i18nunittest/test-packages/blank-yaml-package-tap-i18n/i18n/file2.en.i18n.yml
	deleted:    web.browser/packages/tap_i18n/packages/tap_i18nunittest/test-packages/custom-package-tap-i18n/i18n/bb.i18n.json
	deleted:    web.browser/packages/tap_i18n/packages/tap_i18nunittest/test-packages/custom-package-tap-i18n/i18n/cc-CC.i18n.json
	deleted:    web.browser/packages/tap_i18n/packages/tap_i18nunittest/test-packages/custom-package-tap-i18n/i18n/cc.i18n.json
	deleted:    web.browser/packages/tap_i18n/packages/tap_i18nunittest/test-packages/custom-package-tap-i18n/i18n/file1.en.i18n.json
	deleted:    web.browser/packages/tap_i18n/packages/tap_i18nunittest/test-packages/custom-package-tap-i18n/i18n/file2.en.i18n.json
	deleted:    web.browser/packages/tap_i18n/packages/tap_i18nunittest/test-packages/empty-object-package-tap-i18n/i18n/bb.i18n.json
	deleted:    web.browser/packages/tap_i18n/packages/tap_i18nunittest/test-packages/empty-object-package-tap-i18n/i18n/cc-CC.i18n.json
	deleted:    web.browser/packages/tap_i18n/packages/tap_i18nunittest/test-packages/empty-object-package-tap-i18n/i18n/cc.i18n.json
	deleted:    web.browser/packages/tap_i18n/packages/tap_i18nunittest/test-packages/empty-object-package-tap-i18n/i18n/file1.en.i18n.json
	deleted:    web.browser/packages/tap_i18n/packages/tap_i18nunittest/test-packages/empty-object-package-tap-i18n/i18n/file2.en.i18n.json
	deleted:    web.browser/packages/tap_i18n/packages/tap_i18nunittest/test-packages/pack-with-project-trans-tap-i18n/i18n/bb.i18n.json
	deleted:    web.browser/packages/tap_i18n/packages/tap_i18nunittest/test-packages/pack-with-project-trans-tap-i18n/i18n/cc-CC.i18n.json
	deleted:    web.browser/packages/tap_i18n/packages/tap_i18nunittest/test-packages/pack-with-project-trans-tap-i18n/i18n/cc.i18n.json
	deleted:    web.browser/packages/tap_i18n/packages/tap_i18nunittest/test-packages/pack-with-project-trans-tap-i18n/i18n/file1.en.i18n.json
	deleted:    web.browser/packages/tap_i18n/packages/tap_i18nunittest/test-packages/pack-with-project-trans-tap-i18n/i18n/file2.en.i18n.json
	deleted:    web.browser/packages/tap_i18n/packages/tap_i18nunittest/test-packages/pack-with-project-trans-tap-i18n/i18n/file3.en.i18n.json
	deleted:    web.browser/packages/tap_i18n/packages/tap_i18nunittest/test-packages/project-all-custom-project-tap-i18n/i18n/bb.i18n.json
	deleted:    web.browser/packages/tap_i18n/packages/tap_i18nunittest/test-packages/project-all-custom-project-tap-i18n/i18n/cc-CC.i18n.json
	deleted:    web.browser/packages/tap_i18n/packages/tap_i18nunittest/test-packages/project-all-custom-project-tap-i18n/i18n/file1.en.i18n.json
	deleted:    web.browser/packages/tap_i18n/packages/tap_i18nunittest/test-packages/project-all-custom-project-tap-i18n/i18n/file2.en.i18n.json
	deleted:    web.browser/packages/tap_i18n/packages/tap_i18nunittest/test-packages/project-no-project-tap-i18n/i18n/bb.i18n.json
	deleted:    web.browser/packages/tap_i18n/packages/tap_i18nunittest/test-packages/project-no-project-tap-i18n/i18n/cc-CC.i18n.json
	deleted:    web.browser/packages/tap_i18n/packages/tap_i18nunittest/test-packages/project-no-project-tap-i18n/i18n/file1.en.i18n.json
	deleted:    web.browser/packages/tap_i18n/packages/tap_i18nunittest/test-packages/project-no-project-tap-i18n/i18n/file2.en.i18n.json
	deleted:    web.browser/packages/tap_i18n/packages/tap_i18nunittest/test-packages/project-preload-all-langs/i18n/bb.i18n.json
	deleted:    web.browser/packages/tap_i18n/packages/tap_i18nunittest/test-packages/project-preload-all-langs/i18n/cc-CC.i18n.json
	deleted:    web.browser/packages/tap_i18n/packages/tap_i18nunittest/test-packages/project-preload-all-langs/i18n/file1.en.i18n.json
	deleted:    web.browser/packages/tap_i18n/packages/tap_i18nunittest/test-packages/project-preload-all-langs/i18n/file2.en.i18n.json
	deleted:    web.browser/packages/tap_i18n/packages/tap_i18nunittest/test-packages/project-with-translation-files-under-client-dir-SHOULD-FAIL/client/bb.i18n.json
	deleted:    web.browser/packages/tap_i18n/packages/tap_i18nunittest/test-packages/project-with-translation-files-under-client-dir-SHOULD-FAIL/client/cc-CC.i18n.json
	deleted:    web.browser/packages/tap_i18n/packages/tap_i18nunittest/test-packages/project-with-translation-files-under-client-dir-SHOULD-FAIL/client/file2.en.i18n.json
	deleted:    web.browser/packages/tap_i18n/project-i18n.js
	modified:   web.cordova.json
	deleted:    web.cordova/home/theosp/tap-i18n/unittest/test-packages/blank-package-tap-i18n/package-tap.i18n
	deleted:    web.cordova/home/theosp/tap-i18n/unittest/test-packages/blank-yaml-package-tap-i18n/package-tap.i18n
	deleted:    web.cordova/home/theosp/tap-i18n/unittest/test-packages/custom-package-tap-i18n/package-tap.i18n
	deleted:    web.cordova/home/theosp/tap-i18n/unittest/test-packages/empty-object-package-tap-i18n/package-tap.i18n
	deleted:    web.cordova/home/theosp/tap-i18n/unittest/test-packages/pack-with-project-trans-tap-i18n/package-tap.i18n
	deleted:    web.cordova/home/theosp/tap-i18n/unittest/test-packages/project-preload-all-langs/project-tap.i18n
	deleted:    web.cordova/packages/tap_i18n/packages/tap_i18nunittest/test-lang-files/bb.i18n.json
	deleted:    web.cordova/packages/tap_i18n/packages/tap_i18nunittest/test-lang-files/cc-CC.i18n.json
	deleted:    web.cordova/packages/tap_i18n/packages/tap_i18nunittest/test-lang-files/cc.i18n.json
	deleted:    web.cordova/packages/tap_i18n/packages/tap_i18nunittest/test-lang-files/dd.i18n.json
	deleted:    web.cordova/packages/tap_i18n/packages/tap_i18nunittest/test-lang-files/file1.en.i18n.json
	deleted:    web.cordova/packages/tap_i18n/packages/tap_i18nunittest/test-lang-files/file2.en.i18n.json
	deleted:    web.cordova/packages/tap_i18n/packages/tap_i18nunittest/test-packages/blank-package-tap-i18n/i18n/bb.i18n.json
	deleted:    web.cordova/packages/tap_i18n/packages/tap_i18nunittest/test-packages/blank-package-tap-i18n/i18n/cc-CC.i18n.json
	deleted:    web.cordova/packages/tap_i18n/packages/tap_i18nunittest/test-packages/blank-package-tap-i18n/i18n/cc.i18n.json
	deleted:    web.cordova/packages/tap_i18n/packages/tap_i18nunittest/test-packages/blank-package-tap-i18n/i18n/file1.en.i18n.json
	deleted:    web.cordova/packages/tap_i18n/packages/tap_i18nunittest/test-packages/blank-package-tap-i18n/i18n/file2.en.i18n.json
	deleted:    web.cordova/packages/tap_i18n/packages/tap_i18nunittest/test-packages/blank-yaml-package-tap-i18n/i18n/bb.i18n.yml
	deleted:    web.cordova/packages/tap_i18n/packages/tap_i18nunittest/test-packages/blank-yaml-package-tap-i18n/i18n/cc-CC.i18n.yml
	deleted:    web.cordova/packages/tap_i18n/packages/tap_i18nunittest/test-packages/blank-yaml-package-tap-i18n/i18n/cc.i18n.yml
	deleted:    web.cordova/packages/tap_i18n/packages/tap_i18nunittest/test-packages/blank-yaml-package-tap-i18n/i18n/file1.en.i18n.yml
	deleted:    web.cordova/packages/tap_i18n/packages/tap_i18nunittest/test-packages/blank-yaml-package-tap-i18n/i18n/file2.en.i18n.yml
	deleted:    web.cordova/packages/tap_i18n/packages/tap_i18nunittest/test-packages/custom-package-tap-i18n/i18n/bb.i18n.json
	deleted:    web.cordova/packages/tap_i18n/packages/tap_i18nunittest/test-packages/custom-package-tap-i18n/i18n/cc-CC.i18n.json
	deleted:    web.cordova/packages/tap_i18n/packages/tap_i18nunittest/test-packages/custom-package-tap-i18n/i18n/cc.i18n.json
	deleted:    web.cordova/packages/tap_i18n/packages/tap_i18nunittest/test-packages/custom-package-tap-i18n/i18n/file1.en.i18n.json
	deleted:    web.cordova/packages/tap_i18n/packages/tap_i18nunittest/test-packages/custom-package-tap-i18n/i18n/file2.en.i18n.json
	deleted:    web.cordova/packages/tap_i18n/packages/tap_i18nunittest/test-packages/empty-object-package-tap-i18n/i18n/bb.i18n.json
	deleted:    web.cordova/packages/tap_i18n/packages/tap_i18nunittest/test-packages/empty-object-package-tap-i18n/i18n/cc-CC.i18n.json
	deleted:    web.cordova/packages/tap_i18n/packages/tap_i18nunittest/test-packages/empty-object-package-tap-i18n/i18n/cc.i18n.json
	deleted:    web.cordova/packages/tap_i18n/packages/tap_i18nunittest/test-packages/empty-object-package-tap-i18n/i18n/file1.en.i18n.json
	deleted:    web.cordova/packages/tap_i18n/packages/tap_i18nunittest/test-packages/empty-object-package-tap-i18n/i18n/file2.en.i18n.json
	deleted:    web.cordova/packages/tap_i18n/packages/tap_i18nunittest/test-packages/pack-with-project-trans-tap-i18n/i18n/bb.i18n.json
	deleted:    web.cordova/packages/tap_i18n/packages/tap_i18nunittest/test-packages/pack-with-project-trans-tap-i18n/i18n/cc-CC.i18n.json
	deleted:    web.cordova/packages/tap_i18n/packages/tap_i18nunittest/test-packages/pack-with-project-trans-tap-i18n/i18n/cc.i18n.json
	deleted:    web.cordova/packages/tap_i18n/packages/tap_i18nunittest/test-packages/pack-with-project-trans-tap-i18n/i18n/file1.en.i18n.json
	deleted:    web.cordova/packages/tap_i18n/packages/tap_i18nunittest/test-packages/pack-with-project-trans-tap-i18n/i18n/file2.en.i18n.json
	deleted:    web.cordova/packages/tap_i18n/packages/tap_i18nunittest/test-packages/pack-with-project-trans-tap-i18n/i18n/file3.en.i18n.json
	deleted:    web.cordova/packages/tap_i18n/packages/tap_i18nunittest/test-packages/project-all-custom-project-tap-i18n/i18n/bb.i18n.json
	deleted:    web.cordova/packages/tap_i18n/packages/tap_i18nunittest/test-packages/project-all-custom-project-tap-i18n/i18n/cc-CC.i18n.json
	deleted:    web.cordova/packages/tap_i18n/packages/tap_i18nunittest/test-packages/project-all-custom-project-tap-i18n/i18n/file1.en.i18n.json
	deleted:    web.cordova/packages/tap_i18n/packages/tap_i18nunittest/test-packages/project-all-custom-project-tap-i18n/i18n/file2.en.i18n.json
	deleted:    web.cordova/packages/tap_i18n/packages/tap_i18nunittest/test-packages/project-no-project-tap-i18n/i18n/bb.i18n.json
	deleted:    web.cordova/packages/tap_i18n/packages/tap_i18nunittest/test-packages/project-no-project-tap-i18n/i18n/cc-CC.i18n.json
	deleted:    web.cordova/packages/tap_i18n/packages/tap_i18nunittest/test-packages/project-no-project-tap-i18n/i18n/file1.en.i18n.json
	deleted:    web.cordova/packages/tap_i18n/packages/tap_i18nunittest/test-packages/project-no-project-tap-i18n/i18n/file2.en.i18n.json
	deleted:    web.cordova/packages/tap_i18n/packages/tap_i18nunittest/test-packages/project-preload-all-langs/i18n/bb.i18n.json
	deleted:    web.cordova/packages/tap_i18n/packages/tap_i18nunittest/test-packages/project-preload-all-langs/i18n/cc-CC.i18n.json
	deleted:    web.cordova/packages/tap_i18n/packages/tap_i18nunittest/test-packages/project-preload-all-langs/i18n/file1.en.i18n.json
	deleted:    web.cordova/packages/tap_i18n/packages/tap_i18nunittest/test-packages/project-preload-all-langs/i18n/file2.en.i18n.json
	deleted:    web.cordova/packages/tap_i18n/packages/tap_i18nunittest/test-packages/project-with-translation-files-under-client-dir-SHOULD-FAIL/client/bb.i18n.json
	deleted:    web.cordova/packages/tap_i18n/packages/tap_i18nunittest/test-packages/project-with-translation-files-under-client-dir-SHOULD-FAIL/client/cc-CC.i18n.json
	deleted:    web.cordova/packages/tap_i18n/packages/tap_i18nunittest/test-packages/project-with-translation-files-under-client-dir-SHOULD-FAIL/client/file2.en.i18n.json
	deleted:    web.cordova/packages/tap_i18n/project-i18n.js


no changes added to commit (use "git add" and/or "git commit -a")


```

I probably noticed that the build process included the unittest in the build and got rid of it in v1.8.2 - that's pretty much it. Wouldn't worry about any code you are missing - you can safely work on v1.8.1 .

**Comment by doggy8088** [comment]
When will this project bump version to `v1.8.2`?

**Comment by theosp** [comment]
Today. Better late, then never.
